package.json

```diff
@@ -97,7 +95,7 @@
   "bundlewatch": {
     "files": [
       {
-        "path": "packages/*/build/umd/*.production.js"
+        "path": "packages/*/build/lib/*.js"
```

Comment:
I'm not sure if the report we're getting is that helpful now:

![Screenshot 2023-06-05 at 09 22 19](https://github.com/TanStack/query/assets/1021430/c1ac463f-e115-4a1e-9fd0-37c7e38d9f3a)